app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -54,4 +58,32 @@ class PatientEntryUpdateTest {
             )
         )
   }
+
+  //Tests for validation errors in Patient Entry Screen
+  @Test
+  fun `when the user leaves full name field empty, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.FULL_NAME_EMPTY)
+    val givenModel = defaultModel
+        .fullNameChanged("")
+        .ageChanged("21")
+        .genderChanged(Just(Gender.Male))
+        .phoneNumberChanged("7721084840")
+        .streetAddressChanged("street")
+        .colonyOrVillageChanged("village")
+        .districtChanged("district")
+        .stateChanged("state")
+        .zoneChanged("zone")
+
+    updateSpec
+        .given(givenModel)
+        .`when`(SaveClicked)
```

Comment:
Can we use `whenEvent` instead of `when` please?

Reply:
This applies to all the tests in this PR.

Reply:
Resolved this issue in the next pr